src/uffs/uffs_tree.c

```diff
@@ -570,6 +570,9 @@ static URET _BuildTreeStepOne(uffs_Device *dev)
 
 //	printf("s:%d e:%d\n", dev->par.start, dev->par.end);
 	for (block = dev->par.start; block <= dev->par.end; block++) {
+              if (dev->OnBuildTreeBlockProcessed != NULL) {
```

Comment:
Calling this callback at the beginning of processing is misleading.

Maybe rename it to `OnBuildTreeNewBlock` or `OnBuildTreeBlockProcess`?

Reply:
renamed